server/server.go

```diff
@@ -820,14 +824,16 @@ func DefaultServeConfig() *ServeConfig {
 		n = 2
 	}
 	return &ServeConfig{
-		rsaWorkers:     n,
-		ecdsaWorkers:   n,
-		otherWorkers:   2,
-		limitedWorkers: 0,
-		tcpTimeout:     defaultTCPTimeout,
-		unixTimeout:    defaultUnixTimeout,
-		isLimited:      func(state tls.ConnectionState) (bool, error) { return false, nil },
-		poolSelector:   defaultPoolSelector,
+		rsaWorkers:             n,
+		ecdsaWorkers:           n,
+		otherWorkers:           2,
+		remoteWorkers:          2,
+		limitedWorkers:         0,
+		tcpTimeout:             defaultTCPTimeout,
+		unixTimeout:            defaultUnixTimeout,
+		maxConnPendingRequests: 1024,
```

Comment:
We were supposed to increase this to 16684

Reply:
This change maintains the existing default while making it configurable